template/macros.jinja

```diff
@@ -52,6 +52,15 @@
       tplroot ~ ':tofs:files_switch',
       default_files_switch
   ) %}
+  {#- Lookup files or fallback to source_files parameter #}
+  {%- if not lookup %}
+    src_files = source_files
```

Comment:
Need this `set` in a Jinja block.

Reply:
But based on [the comment below](https://github.com/saltstack-formulas/template-formula/pull/77/files#r265373681), it will be removed anyway.

Reply:
Thanks.